host_provider/providers/base.py

```diff
@@ -90,6 +90,14 @@ def _destroy(self, identifier):
     def _all_node_destroyed(self, group):
         pass
 
+    def get_status(self, host):
+        if self._is_ready(host):
+            return "READY"
```

Comment:
maybe is better return boolean instead string, what u think ?

Reply:
My logic was, maybe someday we need something like "Created" or "Destroyed" than we can change that without a big rest contract chance